src/components/scatterplot/Scatterplot.js

```diff
@@ -27,14 +27,21 @@ const makeDefaultGetCellPosition = mapping => (cellEntry) => {
   return [mappedCell[0], -mappedCell[1], 0];
 };
 const makeDefaultGetCellCoords = mapping => cell => cell.mappings[mapping];
-const makeDefaultGetCellColors = cellColors => cellEntry => (
-  cellColors && cellColors.get(cellEntry[0])
-) || DEFAULT_COLOR;
-const makeDefaultGetCellIsSelected = cellSelection => cellEntry => (
-  cellSelection
-    ? cellSelection.includes(cellEntry[0])
-    : true // If nothing is selected, everything is selected.
-);
+const makeDefaultGetCellColors = (cellColors, cellOpacityScale) => (cellEntry) => {
```

Comment:
In general if possible, should we get rid of the `Scale` suffix?

Reply:
Yes I think that is a good idea, at least for variables that we define